engine/src/main/scala/cromwell/engine/workflow/ShadowWorkflowActor.scala

```diff
@@ -96,7 +96,7 @@ object ShadowWorkflowActor {
 class ShadowWorkflowActor(workflowId: WorkflowId,
                           startMode: StartMode,
                           workflowSources: WorkflowSourceFiles,
-                          conf: Config) extends LoggingFSM[ShadowWorkflowActorState, ShadowWorkflowActorData] with ActorLogging{
+                          conf: Config) extends LoggingFSM[ShadowWorkflowActorState, ShadowWorkflowActorData] with ActorLogging {
```

Comment:
I disagree that "doesn't clean up everything" is a good reason to avoid "clean up one thing", as the former then relies on a clean up PR that's likely to never happen. If you'd prefer I can submit a cleanup specific PR to add that space w/o having it in the midst of other changes.